### PR TITLE
[RHPAM-1629] Added controllerUser and mavenUser where needed

### PR DESCRIPTION
**Pull request by ruromero**
Signed-off-by: Ruben Romero Montes <rromerom@redhat.com>

Fix https://issues.jboss.org/browse/RHPAM-1629

### CODE REVIEW

File: apb.yml [hunk 1]
```diff
@@ -249,15 +265,13 @@ _apb_controller_user: &_apb_controller_user
   required: false
   type: string
   default: monitorUser
-  display_group: Controller Integration
 _apb_controller_pwd: &_apb_controller_pwd
   name: apb_controller_pwd
   title: Controller Password
   description: User password to access the Controller.
```
Comment (jakubschwan) [inline apb.yml]:
```suggestion
   description: User password to access the Controller. Generated if empty
```

Comment (jakubschwan) [reply apb.yml]: Is this suggestion true?

File: apb.yml [hunk 2]
```diff
@@ -249,15 +265,13 @@ _apb_controller_user: &_apb_controller_user
   required: false
   type: string
   default: monitorUser
-  display_group: Controller Integration
```
[the remark this answers is not in the document]
Comment (ruromero) [reply apb.yml]: No, that means that each plan will override the display_group. See below

File: apb.yml (hunk 1 above)
Comment (ruromero) [reply apb.yml]: Yes

**Comment by ruromero** [comment]
Rebased, please review @jakubschwan

**Pull request merged**